### ISSUE TITLE
OC possibility

### ISSUE THREAD

**Issue by CheffOfGames** [dylanpiera/Mafia-Quickplay-DiscordBot#48]
The possibility to start an OC by stating /OC <PLAYER>.
This may create a seperate channel in which those players can talk privately.

**Comment by dylanpiera** [comment]
On discord, you can literally just PM someone though. Besides, you'd either have to PM the bot this command, or everyone would know regardless. What's exactly the use you got planned for this?

**Comment by CheffOfGames** [comment]
in the possibility of there being spectators, you might like it if everything goes automatically

**Comment by dylanpiera** [comment]
Eh. It would involve the creation of a server channel, which could slow down the bot and stuff...

I'm not yet seeing the entire benefit of this. Besides, we haven't decided yet if we wanna make OC a thing in these

**Comment by CheffOfGames** [comment]
true, but if we do it might be easy to do it this way, and i am not taking the bot speed into account here.
